CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# Release 2.27.0 02-06-2021
+
+- b/184731851: Fix overhead latency calculation for backend timeout (#505)
+- b/188906398: Reduce backoff initial latency. (#520)
+- b/188808823: Simplify CORS regex and fix brittle tests (#519)
+- Update release scripts to allow patch release (#518)
+- b/188583118: Update googleapis/googleapis for golang code (#516)
+- Changelog for v2.26.1 (#517)
+- GCS Runner: Handle deadline exceeded properly (#515)
```

Comment:
remove lines that not user facing or feature or bug fixing.  such as. 8, 6, 9

Reply:
Done.